.github/workflows/reusable-agent-build-container-images.yml

```diff
@@ -474,6 +477,10 @@ jobs:
         run: |
           export RETRY_ATTEMPTS="14"
           export SLEEP_DELAY="10"
+          
+          # Wait for at least one hello-printer job to be scheduled and completed. 
+          RETRIES=1; 
+          until echo -n "Waiting for first job to be completed - retry $RETRIES: " && kubectl wait --for=condition=complete job -l app=hello-printer || [ $RETRIES -gt 60 ]; do sleep 1; RETRIES=$(($RETRIES+1)); done
```

Comment:
If the retry limit is hit this won't cause the job to fail.  Instead of a loop couldn't the `kubectl wait` just have 60 sec timeout?

Reply:
Good point. I will add a check and make it fail.
I can't use kubectl wait here, because I am waiting for a job to complete and that job might not even be created yet.
Unfortunatelly there's not better way to do this.